platforms/documentation/docs/src/docs/userguide/best-practices/bp_general.adoc

```diff
@@ -15,6 +15,29 @@
 [[best_practices_general]]
 = General Gradle Best Practices
 
+[[use_kotlin_dsl]]
+== Use Kotlin DSL
+
+Prefer the Kotlin DSL (`build.gradle.kts`) over the traditional Groovy DSL (`build.gradle`) when authoring new builds or creating new subprojects in existing builds.
+
+=== Explanation
+
+The Kotlin DSL offers several advantages over the Groovy DSL:
+
+* Strict typing: IDEs provide better auto-completion and navigation with the Kotlin DSL.
+* Improved readability: Code written in Kotlin is often easier to follow and understand.
+* Single-language stack: Projects that already use Kotlin for production and test code don’t need to introduce Groovy just for the build.
+
+Since Gradle 8.0, link:https://blog.gradle.org/kotlin-dsl-is-now-the-default-for-new-gradle-builds[Kotlin DSL is the default] for new builds created with `gradle init`.
```

Comment:
Do we want to mention that Android Studio defaults to kts as well? 

https://developer.android.com/build/migrate-to-kotlin-dsl#timeline